pytorch/pytorch
# quant docs: clean up hardswish

Stack from [ghstack](https://github.com/ezyang/ghstack):
* #40377 quant docs: add and clean up ELU
* #40346 quant docs: add and clean up BatchNorm{n}d
* #40345 quant docs: add and clean up InstanceNorm{n}d
* #40343 quant docs: add and clean up GroupNorm
* #40342 quant docs: add and clean up LayerNorm
* #40341 quant docs: add and clean up hardtanh
* #40340 quant docs: add and clean up hardsigmoid
* **#40323 quant docs: clean up hardswish**

Cleans up the naming and the function param docs for quantized hardswish.
Remove redundant docstrings and link to floating point modules instead.

Test plan:
* build the docs on Mac OS
* verify that every link works as expected

Differential Revision: [D22152638](https://our.internmc.facebook.com/intern/diff/D22152638)
